1369634: Dont log proxy html errors for hyperv: (ENT-613)

nikosmoum opened the pull request:
- When hyperv gets an HTML page as error response from a proxy, don't log the whole html, but try to scrape the title off of it.
- If scraping the title doesn't work, only log the http error code.

jirihnidek commented on virtwho/virt/hyperv/hyperv.py:
```diff
@@ -395,9 +395,11 @@ def post(self, body):
                 # both old and new namespaces that HyperV uses
                 if errorcode is None or errorcode.text != '2150858778':
-                    self.logger.debug("Invalid response (%d) from Hyper-V: %s", response.status_code, data)
+                    title = xml.find('.//title')
```
I know that you are using existing variable and it's not your fault, but please do small refactoring here and rename `xml` to something like `xml_doc`, because `xml` is well known Python module (`ElementTree` is imported from this module). It could cause some issues in the future.

jirihnidek reviewed (changes requested):
Overall it is good PR. Please do small refactoring of code to avoid future problems.

nikosmoum committed:
1369634: Dont log proxy html errors for hyperv:
- When hyperv gets an HTML page as error response from a proxy, don't log the whole html, but try to scrape the title off of it.
- If scraping the title doesn't work, only log the http error code.
- Changed all variables named 'xml' to 'xml_doc' in hyperv.py to avoid conflict with the new python keyword.

nikosmoum commented:
@jirihnidek fixed all variables in this file called `xml`

jirihnidek reviewed (approved):
ACK
